components-core/src/main/java/com/adyen/checkout/components/model/paymentmethods/PaymentMethod.java

```diff
@@ -37,6 +37,7 @@
     private static final String FUNDING_SOURCE = "fundingSource";
     private static final String ISSUERS = "issuers";
     private static final String CONFIGURATION = "configuration";
+    private static final String DETAILS = "details";
```

Comment:
Maybe add a comment saying this is from the old API versions?

Reply:
👍